packages/gatsby-theme-api-docs/package.json

```diff
@@ -32,15 +32,15 @@
     "prop-types": "15.7.2"
   },
   "devDependencies": {
-    "@commercetools-docs/gatsby-theme-docs": "14.0.7",
+    "@commercetools-docs/gatsby-theme-docs": "15.0.0",
     "@commercetools-docs/rmf-codegen": "12.1.0",
     "gatsby": "3.0.4",
     "gatsby-source-filesystem": "3.0.0",
     "react": "17.0.1",
     "react-dom": "17.0.1"
   },
   "peerDependencies": {
-    "@commercetools-docs/gatsby-theme-docs": "14.x",
+    "@commercetools-docs/gatsby-theme-docs": "15.0.0",
```

Comment:
We should use `15.x` here...I'll change it before releasing.

Reply:
On a second thought, we can just keep it like this, as it's managed by Changeset anyway.